hrmcalcs2oo.py

```diff
@@ -0,0 +1,54 @@
+class hrmcalcs:
```

Comment:
nit: should be CapitalCamelCase: `HrmCalcs`

Reply:
Or consider naming it as an entity or object: an `HrmCalculator`

Reply:
You could _consider_ refactoring things into a general `HeartRateMonitor` class that can hold some HR data and then also have associated methods (that are here) to perform calculations on the current `HeartRateMonitor`'s stored data.

Reply:
So I think that was our original plan but we then split it up into separate files so that we could work separately and merge separate changes through GitHub. The only way I see now to do this is to copy and paste things together. What would have been the ideal workflow to work toward this from the beginning?

Reply:
Ah I see! In the future, even if people are working in the same file but only making changes in separate functions, you should be able to merge that together in github (or do a rebase before merging into master if someone else has made changes in that file on master). I know it's a little scary to be working in the same file, but the auto merging should work without conflicts if editing is localized to different functions in the same file :).